modules/develop/pages/data-transforms/build.adoc

```diff
@@ -338,14 +407,14 @@ import (
 
 func main() {
   client := sr.NewClient()
```

Comment:
Think we want the client to be global here, like this:

https://github.com/redpanda-data/redpanda/blob/c7e27f720856b1787e4b26164863c3f7597acc1f/src/transform-sdk/go/transform/internal/testdata/schema-registry/transform.go#L26-L33